mqtt/mqtt-edgehub/tests/common/mod.rs

```diff
@@ -43,6 +40,24 @@ where
             self.0.authorize(activity)
         }
     }
+
+    fn update(&mut self, update: Box<dyn Any>) -> Result<(), Self::Error> {
+        self.0.update(update)
+    }
+}
+
+pub struct BottomLevelDummyAuthorizer;
+
+impl Authorizer for BottomLevelDummyAuthorizer {
```

Comment:
shouldn't this guy to implement `fn update` ?

Reply:
It doesn't need to since it's never called, so it just takes the default, but doesn't matter anymore since I've deleted it.

Reply:
ah, I missed the default implementation